src/test/java/it/gov/pagopa/arc/utils/TestUtils.java

```diff
@@ -59,4 +63,25 @@ public static KeyPair genKeyPair() throws NoSuchAlgorithmException {
         return keyGen.generateKeyPair();
     }
 
+    public static void assertNotNullFields(Object dto, String... excludedFields) {
+        Class<?> clazz = dto.getClass();
+        Set<String> excludedFieldsSet = new HashSet<>(Arrays.asList(excludedFields));
+
+        Arrays.stream(clazz.getDeclaredFields())
```

Comment:
declaredFields will ignore inherithed fields, could you use spring utility ReflectionUtils as done in
https://github.com/pagopa/idpay-payment/blob/main/src/test/java/it/gov/pagopa/common/utils/TestUtils.java#L45
?